src/Parser/DecimalMoneyParser.php

```diff
@@ -16,7 +14,7 @@
  */
 final class DecimalMoneyParser implements MoneyParser
 {
-    const DECIMAL_PATTERN = '/^(?P<sign>-)?(?P<digits>0|[1-9]\d*)?\.?(?P<fraction>\d+)?$/';
```

Comment:
Note that removing this constant is probably a BC break.

Reply:
reverted